### ISSUE TITLE
one-sided geom_sina

### ISSUE THREAD

**Issue by luisDVA** [thomasp85/ggforce#110]
Hi Thomas,
I tried to implement one-sided sina plots by changing the binning functions but I couldn't get it to work. Is this something that can already be done through adjustments to stat_sina parameters?

**Comment by sidiropoulos** [comment]
Hi @luisDVA, 
Can you elaborate a bit on what are you trying to achieve?

**Comment by luisDVA** [comment]
sure,
I was trying to shift the points along the x axis to only one direction relative to the value of x (an absolute value derived from the local density along the y axis).

Below I covered half the points in a sina plot with white rectangles, but this should explain my idea a little better, which is to shift all the points to either the right (+) or left (-) of the reference value.

![sina_right](https://user-images.githubusercontent.com/5580938/48281791-75c4d180-e436-11e8-84e2-fde8491fbcd3.png)


thope this helps!

**Comment by sidiropoulos** [comment]
Thanks! 
Just to be clear, by "shift the points" you mean that you only want the points to be plotted on either the left or the right side of the corresponding x-axis label/coordinate and not to also "slide" them away from the label. 

Either way, the feature is not available but I'm working on implementing the example you posted above.

**Comment by luisDVA** [comment]
that's correct, just plotting the points on one side of the x-axis coordinate. Thanks for correcting the terminology and for looking into the issue!

**Comment by jwilmer** [comment]
Just curious if this feature is still a possibility? I think it would be a good one to have.
